src/components/AddPublicationModal/PublicationUpload.js

```diff
@@ -0,0 +1,44 @@
+import React from "react";
+
+const PublicationUpload = props => {
+  return (
+    <div>
+      <label htmlFor="file" className="ui icon" style={{ cursor: "pointer" }}>
+        <span style={styles.uploadText}>Upload an analysis document</span>
```

Comment:
Could the “analysis” stage mention be extracted such that the correct stage name is supplied to PublicationUpload instead?

Reply:
Done